Kloudy/Kloudy/View/CheckLocation/LocationSelectionCollectionViewCell/LocationSelectionCollectionViewCell.swift

```diff
@@ -58,22 +61,22 @@ class LocationSelectionCollectionViewCell: UICollectionViewCell {
     private func setLayOut() {
         locationNameLabel.snp.makeConstraints {
             $0.leading.equalTo(20)
-            $0.top.bottom.equalTo(36)
+            $0.centerY.equalToSuperview()
         }
         
         weatherImage.snp.makeConstraints {
             $0.leading.equalTo(152)
-            $0.top.bottom.equalTo(35)
+            $0.centerY.equalToSuperview()
         }
         
         temperatureLabel.snp.makeConstraints {
-            $0.trailing.equalTo(20)
-            $0.top.equalTo(40)
+            $0.trailing.equalTo(-20)
+            $0.centerY.equalToSuperview()
         }
         
         diurnalTemperatureLabel.snp.makeConstraints {
-            $0.trailing.equalTo(20)
-            $0.top.bottom.equalTo(26)
+            $0.trailing.equalTo(-88)
```

Comment:
```suggestion
            $0.trailing.equalToSuperView().inset(88)
```
위처럼 변경한다면 SnapKit 코드를 더욱 직관적으로 이해할 수 있을 것 같아요!